pkg/kubectl/cmd/resource/get_test.go

```diff
@@ -246,7 +244,8 @@ func TestGetUnknownSchemaObject(t *testing.T) {
 	cmd.Run(cmd, []string{"type", "foo"})
 
 	expected := []runtime.Object{cmdtesting.NewInternalType("", "", "foo")}
-	actual := tf.Printer.(*testPrinter).Objects
+	actual := []runtime.Object{}
+	//actual := tf.Printer.(*testPrinter).Objects
```

Comment:
Unused, remove?

Reply:
> Unused, remove?

History in case someone wonders how it "worked" before even though the test is flawed.